lib/jobs/sftp-job.js

```diff
@@ -69,14 +73,54 @@ function sftpJobFactory(
             ssh.on('ready', function() {
                 ssh.sftp(function(err, sftp) {
                     if (err) { reject(err); }
-                    sftp.fastPut(
+                    if (self.isPDU === "true") {
+                        logger.info("This is an iPDU, isPDU:", self.isPDU);
```

Comment:
There are two much logger messages, actually this is not necessary. Usually we only log important millstones inside a task.